sysboundary/ionosphere.cpp

```diff
@@ -768,61 +969,66 @@ namespace SBC {
       };
 
 
-      // Trace node coordinates outwards until a non-sysboundary cell is encountered
-      //#pragma omp parallel for
-      for(uint n=0; n<nodes.size(); n++) {
+      // Pick an initial stepsize
+      Real stepSize = min(100e3, technicalGrid.DX / 2.); 
+      
+      #pragma omp parallel firstprivate(stepSize)
```

Comment:
@ursg   Is this thread safe? I am initializing stepSize above as firstprivate and then I am starting a parallel region. In the parallel for below each thread should modify its' own stepSize accordingly.

Reply:
Can't you just declare it inside the parallel region? Then it is a variable private to the thread and I don't need to look up `firstprivate`. :)